src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/HtmlHelperFormExtensions.cs

```diff
@@ -140,9 +142,11 @@ public static MvcForm BeginForm(this IHtmlHelper htmlHelper, FormMethod method)
         /// </summary>
         /// <param name="htmlHelper">The <see cref="IHtmlHelper"/> instance this method extends.</param>
         /// <param name="method">The HTTP method for processing the form, either GET or POST.</param>
-        /// <param name="suppressAntiforgery">
-        /// If <c>true</c>, suppresses the generation an &lt;input&gt; of type "hidden" with an antiforgery token. By
-        /// default &lt;form&gt; elements will automatically include an antiforgery token.
+        /// <param name="antiforgery">
```

Comment:
Ditto all doc comment comments for the other `<param name="antiforgery">`s. They should all be identical when you're done.